Packages/com.unity.inputsystem/InputSystem/Controls/InputControl.cs

```diff
@@ -195,10 +195,10 @@ public bool noisy
             get { return InputControlPath.TryFindChild(this, path); }
         }
 
-        public Type valueType
-        {
-            get { return m_ValueType; }
-        }
+        /// <summary>
+        /// Returns the underlying value type of this control.
```

Comment:
For doc comments in general, please go beyond the obvious. For this one, the fact it can be null is undocumented, for example. Also noteworthy that it's not necessarily the same as `ReadValueAsObject().GetType()`. Etc.